### DISCUSSION TITLE
Continue execution from outside Ray after ray()->pause();

### DISCUSSION THREAD

**Discussion by Patabugen** [spatie/laravel-ray#342] [Q&A]
This might be a bit of a niche request - I use a Streamdeck while I develop for some handy shortcuts.

It'd be great to be able to "Continue" when I've called `ray()->pause()` programmatically - for example with a keyboard shortcut (which can be emulated) or even better with some sort of CLI call to Ray.

Is there any way to Continue without using the mouse?

![image](https://github.com/spatie/laravel-ray/assets/196795/4197f562-c974-4c5e-9919-6c3db214774c)

**Comment by patinthehat** [comment]
@Patabugen It's definitely possible, but it wouldn't be too pretty. 

Output the UUID returned by the `ray` call to a file, then read that in from a script and send the `continue/stop` request using whatever scripting language you prefer.
